.github/workflows/sync-translations.yml

```
@@ -0,0 +1,134 @@
+name: Migrate translations from the old Transifex project
+
+on:
+  workflow_dispatch:
+
+
+jobs:
+  migrate-translations:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        batch:
```

naming nit: it's not clear what `batch` means in this context. I don't have a suggestion for a name that I love, but we can probably go for something that matches up with the naming in Transifex.

```suggestion
        resource:
```

---

done